### ISSUE TITLE
Seems to not work on safari

### ISSUE THREAD

**Issue by iresharma** [Glitchii/embedbuilder#16]
<img width="1792" alt="Screenshot 2022-03-15 at 2 33 08 AM" src="https://user-images.githubusercontent.com/32684272/158261051-d59c6ee7-0d9e-41c7-8d5a-84f135389ae9.png">

But it seems to be working perfectly on chrome 
<img width="1792" alt="Screenshot 2022-03-15 at 2 39 20 AM" src="https://user-images.githubusercontent.com/32684272/158261845-b5510ea6-414b-463b-82db-214f2ed5fba7.png">

**Comment by Glitchii** [comment]
Yes, this is a known issue. It is not working because Safari currently does not support regex look-behind or look-ahead (which is very surprising and disappointing for a big browser like Safari). I added this issue to the projects board (https://github.com/Glitchii/embedbuilder/projects/3).

Currently, I don't have a plan in mind of how I can by-pass this but it'll be fixed latter
